src/rpc/blockchain.cpp

```diff
@@ -766,14 +766,7 @@ static RPCHelpMan getblock()
 {
     uint256 hash(ParseHashV(request.params[0], "blockhash"));
 
-    int verbosity = 1;
-    if (!request.params[1].isNull()) {
-        if (request.params[1].isBool()) {
-            verbosity = request.params[1].get_bool() ? 1 : 0;
-        } else {
-            verbosity = request.params[1].getInt<int>();
-        }
-    }
+    int verbosity{ParseVerbosity(request.params[1], 1)};
```

Comment:
an annotation can help here
```suggestion
   int verbosity{ParseVerbosity(request.params[1], /*default_verbosity=*/1)};
```

Reply:
Agreed. Updated